managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -4232,4 +4232,24 @@ public void testNoCleanupOffloadLedgerWhenMetadataExceptionHappens() throws Exce
         verify(ledgerOffloader, times(0))
             .deleteOffloaded(eq(ledgerInfo.getLedgerId()), any(), anyMap());
     }
+
+    @Test
+    public void testDeleteCurrentLedgerWhenItIsClosed() throws Exception {
+        ManagedLedgerConfig config = spy(new ManagedLedgerConfig());
+        ManagedLedgerImpl ml = spy((ManagedLedgerImpl) factory.open("testDeleteCurrentLedgerWhenItIsClosed", config));
```

Comment:
Any reason to use `spy()` here? You can set any configurations you want via a new ManagedLedgerConfig and create a new Ledger with the config instance.